cmd/ops/deprecation.go

```diff
@@ -18,14 +18,16 @@ var deprecationsCmd = &cobra.Command{
 Find deprecated fields in queries and mutations given a list of files
 
 The "queries" argument is a file glob matching one or more graphql query or mutation files.`,
-	Args: ExactArgs(1, "you must specify at least one file with queries or mutations"),
+	Args: MinimumNArgs(1, "you must specify at least one file with queries or mutations"),
 	RunE: deprecationsCmdRun,
 }
 
 func init() {
 	Program.AddCommand(deprecationsCmd)
 	deprecationsCmd.Flags().StringVar(&flags.schemaFile, schemaFileFlagName, "", "Server's schema as file or url (required)")
 	deprecationsCmd.MarkFlagRequired(schemaFileFlagName) //nolint:errcheck // will err if flag doesn't exist
+
+	deprecationsCmd.Flags().StringArrayVar(&flags.ignore, ignoreFlagName, []string{}, "Files to ignore (as file blob)")
```

Comment:
Not quite sure what "as a blob" means here...? 🤔 I think maybe "Files to ignore" is enough...

Reply:
ups, sorry, I meant "glob" 😄